db/seeds.rb

```diff
@@ -18,14 +18,15 @@
 Genre.create!(name: "Technology")
 
 user_list = [
```

Comment:
You'll need to update these seeded users to remove the role (see previous comments)